### ISSUE TITLE
GUI delete button is greyed out and does not function (flatpak)

### ISSUE THREAD

**Issue by caj411** [qarmin/czkawka#1117]
**Desktop **

- Czkawka version<Flatpak 6.1.0 gui, installed using pamac>
- OS version<Manjaro 23.0.4 using Gnome 44.5>
- Terminal output:  flatpak run --env=RUST_LOG=debug com.github.qarmin.czkawk        ✔ 
error: app/com.github.qarmin.czkawk/x86_64/master not installed

**Bug Description**

Appears the same or similar issue as reported in report #1104 and #1105.
The delete button is grey and even when I select a file it does not become active, it remains grey.
Basically, the delete button is not functional.
...

**Comment by gideion** [comment]
I am having the same issue, the delete button does not work at all.

Ubuntu 22.04
Flatpak 6.1.0 gui

user:~$ flatpak run --env=RUST_LOG=debug com.github.qarmin.czkawk
error: app/com.github.qarmin.czkawk/x86_64/master not installed

**Comment by caj411** [comment]
It was not the application causing the issue.

**Comment by gideion** [comment]
What is causing the issue?

Why wouldn't the files delete?

**Comment by caj411** [comment]
It was user error, ie it was my fault, but.... I thought I had selected files to delete because the entire line was highlighted, I did not realize that I had to check the box on the left. Here is the but.... When I check the box of the file I want to delete, the delete button is still greyed out, so there is no indication that a file has been selected for deletion. If I press the greyed out delete button it will delete the selected file. Is the fact that the button stays greyed out a bug or just an oversight? On Dec 29, 2023, at 7:06 PM, gideion ***@***.***> wrote: What is causing the issue? Why wouldn't the files delete? — Reply to this email directly, view it on GitHub , or unsubscribe . You are receiving this because you modified the open/close state. Message ID: ***@***.***>

**Comment by caj411** [comment]
I apologize, but I checked further and apparently It is only appearing to delete the files. I tried deleting duplicates and everything appears to work, but it actually has done nothing. The files disappear from the que, but are not being actually deleted. I closed the application then reopened it and ran the same query and the files were still there. I went and double checked and the files were indeed still there, so the application is failing to delete files. On Dec 29, 2023, at 7:06 PM, gideion ***@***.***> wrote: What is causing the issue? Why wouldn't the files delete? — Reply to this email directly, view it on GitHub , or unsubscribe . You are receiving this because you modified the open/close state. Message ID: ***@***.***>

**Comment by gideion** [comment]
Yes, this is the same behavior that i am experiencing when I try to delete files. They appear to delete in the program, but they do not actually get deleated.

**Comment by caj411** [comment]
Do I need to change the status from closed back to open? Or create another bug report?

**Comment by caj411** [comment]
I have reopened the issue since we are both in agreement, something isn't working properly.

**Comment by caj411** [comment]
Thank you for continuing to work on this issue.

**Comment by qarmin** [comment]
Portal based removal was added some time ago, so it should work now.
I don’t plan to release a new Flatpak version of the app, but if anyone is willing to maintain it, feel free to do so.
